contracts/settling_game/library/library_resources.cairo

```diff
@@ -157,27 +208,31 @@ namespace Resources:
     ):
         alloc_locals
 
-        let (r_1_output) = _calculate_resource_output(happiness)
-        let (r_2_output) = _calculate_resource_output(happiness)
-        let (r_3_output) = _calculate_resource_output(happiness)
-        let (r_4_output) = _calculate_resource_output(happiness)
-        let (r_5_output) = _calculate_resource_output(happiness)
-        let (r_6_output) = _calculate_resource_output(happiness)
-        let (r_7_output) = _calculate_resource_output(happiness)
+        let (r_1_output) = _calculate_resource_output(workhuts, happiness)
```

Comment:
it just calculates the amount per resource. Since there are between 1-7 resources on each realm, it calculates all of them. It could probably be more efficient to check in a previous function

Reply:
Unless my eyes are not working correctly, It's calling the same function with the same two args seven times. It's missing the "per resource" thing.

Reply:
-_- 

Updating - this was a relic from the past refactor when the resources has levels...